apps/server/src/modules/deletion/deletion-api.module.ts

```diff
@@ -1,28 +1,30 @@
 import { Module } from '@nestjs/common';
 import { DeletionModule } from '@modules/deletion';
-import { AccountModule } from '@modules/account';
-import { ClassModule } from '@modules/class';
-import { LearnroomModule } from '@modules/learnroom';
-import { FilesModule } from '@modules/files';
-import { PseudonymModule } from '@modules/pseudonym';
-import { LessonModule } from '@modules/lesson';
-import { TeamsModule } from '@modules/teams';
-import { UserModule } from '@modules/user';
 import { LoggerModule } from '@src/core/logger';
 import { AuthenticationModule } from '@modules/authentication';
-import { RocketChatUserModule } from '@modules/rocketchat-user';
+import { CqrsModule } from '@nestjs/cqrs';
 import { Configuration } from '@hpi-schul-cloud/commons';
-import { RocketChatModule } from '@modules/rocketchat';
-import { RegistrationPinModule } from '@modules/registration-pin';
-import { TaskModule } from '@modules/task';
-import { FilesStorageClientModule } from '@modules/files-storage-client';
 import { DeletionRequestsController } from './controller/deletion-requests.controller';
 import { DeletionExecutionsController } from './controller/deletion-executions.controller';
 import { DeletionRequestUc } from './uc';
+import { LearnroomModule } from '../learnroom';
```

Comment:
All the imports must be switched back to @modules/* . Relative pathes are only allowed inside the same module.

But the goal is, to remove them all from the module correct and for account module it is already done and it can be removed? Otherwise we implement a dependency cycle on the most critical part -> accountModule -> AuthorisationModule -> important in nearly every other module.

Reply:
The important point is not solved. Please remove the imports that not nessasary anymore.

Reply:
@CeEv could you clarify it?